src/DependencyInjection/SonataArticleExtension.php

```diff
@@ -41,7 +44,18 @@ public function load(array $configs, ContainerBuilder $container): void
         }
 
         $this->registerParameters($container, $config);
-        $this->registerDoctrineMapping($config);
+
+        if (isset($bundles['SonataDoctrineBundle'])) {
```

Comment:
The dependency is required, but the check is here assumes it is an optional dependency. Maybe we should throw an Exception, if there is no `SonataDoctrineBundle` and no `SonataCoreBundle` registered.

Reply:
I think the deprecation message should only be about EasyExtends and SonataDoctrineBundle, otherwise it could be confusing on how to proceed. You only need to take care of removing easy extends and enabling SonataDoctrineBundle if you want this deprecation to go away.

Reply:
@jordisala1991 please refrain from resolving conversations you don't open, it makes it harder for other contributors to spot such conversations and decide who they think is correct. Now I'm unsure if I should dismiss @core23 's review or not…

@core23 a dependency can be required, but in the case of a bundle it does not mean it will be registered, although it's unlikely.

> Maybe we should throw an Exception, if there is no SonataDoctrineBundle and no SonataCoreBundle registered.

Did you mean "or" instead of "and". BTW, isn't the core bundle deprecated and no longer supposed to be used?

> You only need to take care of removing easy extends and enabling SonataDoctrineBundle if you want this deprecation to go away.

What happens currently if the core bundle isn't registered?

Reply:
AFAIK there is no relation between the SonataDoctrineBundle and the CoreBundle. If you dont register CoreBundle you won't get any error related to this PR.

You could have errors if you don't register CoreBundle and don't register FormExtension / TwigExtension, but that's unrelated

Reply:
And I don't think we should throw an error on stable branch if the user is not registering any of: EasyExtendsBundle and SonataDoctrineBundle, because that could cause errors where before there were none. I left a comment to throw an error on master branch.

Reply:
Sorry for the resolving conversations thing @greg0ire , didnt want to cause troubles

Reply:
Hey, don't worry mate! And thanks for the huge work BTW!